libkbfs/key_bundle_v2.go

```diff
@@ -14,21 +14,21 @@ import (
 	"github.com/pkg/errors"
 )
 
-type ePubKeyTypeV2 int
+type ePubKeyLocationV2 int
```

Comment:
Changed this to "key location", since a writer key can still be in the readers array